userdocs/diagnostics/property-wrapper-requirements.md

```diff
@@ -3,4 +3,6 @@ If a type is marked with the `@propertyWrapper` attribute, it must meet certain
 
 First, all property wrapper types must have a property named `wrappedValue`. This property cannot be static and must have the same access level as the property wrapper type. If the property wrapper provides a `projectedValue` property, it is subject to the same requirements.
 
-Second, none of a property wrapper's initializers may be failable. Additionally, if a property wrapper initializer has a `wrappedValue` parameter, the type of that parameter must either be the same as the type of the `wrappedValue` property or an `@autoclosure` of that type.
\ No newline at end of file
+Second, none of a property wrapper's initializers may be failable. Additionally, if a property wrapper initializer has a `wrappedValue` parameter, the type of that parameter must either be the same as the type of the `wrappedValue` property or an `@autoclosure` of that type.
+
+To understand property wrappers in more detail check out [Propeties](https://docs.swift.org/swift-book/LanguageGuide/Properties.html) section in _The Swift Programming Language_.
```

Comment:
maybe link to the property wrapper section directly? https://docs.swift.org/swift-book/LanguageGuide/Properties.html#ID617